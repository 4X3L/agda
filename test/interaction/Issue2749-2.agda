module Issue2749-2 where

-- testing unicode lambda and arrow
id : {A : Set} -> A -> A
id = {!!}

-- testing unicode double braces
it : {A : Set} {{a : A}} → A → A
it = {!!}

data B : Set where
  mkB : B → B → B

-- testing unicode suffixes
left : B → B
left b₁ = {!!}
