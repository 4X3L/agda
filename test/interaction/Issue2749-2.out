(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : .A → .A ?1 : .A → .A ?2 : B " nil)
((last . 1) . (agda2-goals-action '(0 1 2)))
(agda2-give-action 0 "λ x → ?")
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?1 : .A → .A ?2 : B ?3 : .A " nil)
((last . 1) . (agda2-goals-action '(3 1 2)))
(agda2-status-action "ShowImplicit")
((last . 1) . (agda2-goals-action '(3 1 2)))
((last . 2) . (agda2-make-case-action '("it {A} ⦃ a ⦄ x = ?")))
((last . 1) . (agda2-goals-action '(3 1 2)))
(agda2-status-action "")
((last . 1) . (agda2-goals-action '(3 1 2)))
((last . 2) . (agda2-make-case-action '("left (mkB b₁ b₂) = ?")))
((last . 1) . (agda2-goals-action '(3 1 2)))
